optaplanner-examples/src/test/resources/org/optaplanner/examples/tsp/data/circle_25.xml

```diff
@@ -0,0 +1,235 @@
+<TspSolution id="1">
```

Comment:
I think it would be better to keep the testing data in the same package as the test that uses it, e.g. `org.optaplanner.examples.lowentitycount`. It makes it easier to find classes and resources that are intended to work together.

Reply:
That's right, on the other hand, it might be used by multiple tests from different packages. But this dataset is probably going to be replaced by europe40.xml (or other). See the next comment of this file.